packages/account/src/Components/trading-assessment/risk-tolerance-warning-modal.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Button, DesktopWrapper, Icon, MobileDialog, MobileWrapper, Modal, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const RiskToleranceWarningModal = ({ show_risk_modal, onClick, title, button_text, body_content }) => (
+    <React.Fragment>
+        <MobileWrapper>
+            <MobileDialog
+                portal_element_id='modal_root'
+                visible={show_risk_modal}
+                title={title}
+                wrapper_classname='risk-acceptance'
+                has_close_icon={false}
+            >
+                <Icon icon='IcRedWarning' size={65} />
+                <Text as='p' size='xs' align='center' line_height='l'>
+                    {body_content}
+                </Text>
+                <Button type='button' large text={button_text ?? localize('OK')} primary onClick={onClick} />
```

Comment:
do we need here '??'
it means that we could pass 0 to button_text. would it be appropriate?
maybe it is better to use here '||'